nautobot/docs/user-guide/administration/guides/prometheus-metrics.md

```diff
@@ -25,6 +25,11 @@ CACHES = {
 }
 ```
 
+<!-- TODO: Which version? -->
+++ 2.2.0
```

Comment:
I'm thinking 2.2.1 is fine for this? It's not a breaking change and not significant enough to push out to 2.3.0.

```suggestion
+++ 2.2.1
```